network/types.go

```diff
@@ -111,4 +111,8 @@ type Config struct {
 	// A zero value falls back to runenv.TestInstanceCount (i.e. all instances
 	// participating in the test run).
 	CallbackTarget int `json:"-"`
+
+	// WhitelistAll whitelists all traffic through the data plane (except for the
+	// control traffic).
+	WhitelistAll bool
```

Comment:
Hm, how about we switch this to an enum field `RoutingPolicy`? That would be more future-proof.

```go
type RoutingPolicyType string

const (
    WhitelistAll = RoutingPolicyType("whitelist_all")
)

type Config struct {
    ...
    
    RoutingPolicy RoutingPolicyType
}
```

Reply:
Maybe use `AllowList` and `DenyList` (or `BlockList`) - it sounds similar to iptables terminology.

Reply:
Oh yeah. There terms are so embedded in our ways... Let's avoid controversial terminology. `allow_all` and `deny_all` (future) are the right ways to go here.